backend/dataall/modules/dataset_sharing/services/dataset_alarm_service.py

```diff
@@ -19,11 +19,9 @@ def trigger_table_sharing_failure_alarm(
             target_environment: Environment,
     ):
         log.info('Triggering share failure alarm...')
-        subject = (
-            f'ALARM: DATAALL Table {table.GlueTableName} Sharing Failure Notification'
-        )
+        subject = f'Data.all Share Failure for Table {table.GlueTableName}'[:100]
```

Comment:
lowercase data.all

Reply:
do not capitalize words because the subjects are inconsistent.. for example in one subject word 'permissions' is not capitalized.. I'd just keep it normal english and only capitalize the first letter but because data.all always starts with lower d I'd just go with:

f'data.all share failure for table {table.GlueTableName}'[:100]

Reply:
I wanted to be consistent with OS original Camel case usage in Subject. 
I dont mind changing the lowercase permissions to Permissions to be consistent across all Subjects in the alarms.

Additionally, I just did a quick search with case type match across all the code. 
Data.all is clearly the most used option over data.all.
data.all is only used in logs in quicksight.py. Everywhere else, Data.all is used. To be consistent across most files, I prefer using Data.all

Reply:
Sounds good, if that's what the project is using then i'm OK with it.. I assumed they would use data.all. we can resolve.